assets/js/pages/HostsList/HostsList.test.jsx

```diff
@@ -121,6 +125,38 @@ describe('HostsLists component', () => {
         ).toBeVisible()
       );
     });
+
+    it('should show only unique SIDs', async () => {
+      const host = hostFactory.build();
+      const duplicateSID = faker.string.alpha({ casing: 'upper', count: 3 });
+      const id = faker.string.uuid();
+      const sapInstances = sapSystemApplicationInstanceFactory.buildList(2, {
+        sap_system_id: id,
+        sid: duplicateSID,
+        host_id: host.id,
+      });
+      const databaseInstances = databaseInstanceFactory.buildList(2, {
+        database_id: id,
+        host_id: host.id,
+        sid: duplicateSID,
+      });
+      const state = {
+        ...defaultInitialState,
+        hostsList: {
+          hosts: [host],
+        },
+        sapSystemsList: {
+          applicationInstances: [...sapInstances],
```

Comment:
you don't need to expand values in a list, it is already a list hehe
`applicationInstances: sapInstances` (i would call it as it is in fact, applicationInstances).
The same with `databaseInstances. 

In fact, most probably, you can just remove them after the `:`, as they have the same name, and they should be just used that way.
Give a try (it might be prettied by the formatter)

```
sapSystemsList: { applicationInstances },
databasesList: { databaseInstances },
```